vhdbuilder/scripts/linux/mariner/tool_installs_mariner.sh

```diff
@@ -114,3 +114,11 @@ EOF
 # Future base images will already have rsyslog installed with 755 /etc/rsyslog.d
     chmod 755 /etc/rsyslog.d
 }
+
+enableMarinerKata() {
+    # Enable the mshv boot path
+    sudo sed -i -e 's@menuentry "CBL-Mariner"@menuentry "Dom0" {\n    search --no-floppy --set=root --file /EFI/Microsoft/Boot/bootmgfw.efi\n        chainloader /EFI/Microsoft/Boot/bootmgfw.efi\n}\n\nmenuentry "CBL-Mariner"@'  /boot/grub2/grub.cfg
```

Comment:
this seems...resilient...

is there really no better way than sed'ing it? you can't write the whole file out?

whitespace/newlines feels like a recipe for breakage

Reply:
I'm not so thrilled about such an unwieldy sed either, but since it's what we've been building with for a few months I'd rather not change it at the last minute before we get a few images out.

This line gets removed when we've updated the boot path to fall back to defaults on a non-nested VM SKU.

Alternatively, @neha170 could we look into having both regular and DOM0 menu entries in our base image then just setting something like "GRUB_DEFAULT=1" as we're customizing the image in AgentBaker?

Reply:
np, can leave as is